### ISSUE TITLE
Problem importing the ecoinvent3.10 database

### ISSUE THREAD

**Issue by ommeh123** [LCA-ActivityBrowser/activity-browser#1342]
### Issue

Hello
I am experiencing an issue while trying to import the ecoinvent 3.10 database. When I enter my username and password, I receive an 'unknown connection error.' Could you please guide me on how to successfully import this database into AB?

**Comment by mrvisscher** [comment]
Hi @ommeh123,

Is there any more information on the error in the console?

Kind regards,

Marin

**Comment by ommeh123** [comment]
Hi Marin,

Thank you for your response. I’ve checked the console, and the error
message is as follows:

[image: image.png]

[image: image.png]



Thank you for your assistance.

Kind regards,

 Hani

‫‪Marin Visscher‬‏ ***@***.***‬‏> در تاریخ جمعه ۱۶ اوت ۲۰۲۴
ساعت ۱۰:۵۱ نوشت:‬

**Comment by mrvisscher** [comment]
Hi Hani,

I'm afraid the screenshots you've sent don't show up on GitHub. Please try again, or send them to me personally via: m.r.visscher@cml.leidenuniv.nl 

Kind regards,

Marin

**Comment by SaraGo97** [comment]
Hi, I am also experiencing issues importing the EI database. It seems that the educational EI license I have does not allow me to download files (e.g. ecospold files). Were you also facing a similar issue?

**Comment by mrvisscher** [comment]
Hi @SaraGo97 ,

Do you get an error message of sorts in the console?

**Comment by SaraGo97** [comment]
Yes, if clicking on "import remote data", the username and password of my faculty license are not recognized althought the license is still valid

**Comment by mrvisscher** [comment]
What does the consol/terminal say when you try to login? Can you check whether you can still login here: https://ecoquery.ecoinvent.org/

**Comment by SaraGo97** [comment]
If I click on "Import remote data (download)", the system says "invalid username/password" but I checked with the EI customers assistance and they said I should ask Brightway for support. I thought of trying texting here... Yes, I can still login to ecoinvent, the license is still valid but does not allow me to download ecospold files. As the "import data from remote" also does not seem possible, I am stuck...
